### PR TITLE
Updated packages ownership

### PR DESCRIPTION
**Pull request by codebien**
I missed them in #2466

### CODE REVIEW

**Review by na-- — DISMISSED** [review]

File: packaging/rpm.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "k6",
     "Summary": "Load testing for the 21st century",
     "description": "k6 is a developer centric open source load testing tool for testing the performance of your backend infrastructure. It’s built with Go and JavaScript to integrate well into your development workflow, so you can stay on top of performance without fuzz.",
-    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Load Impact \n - Go to the release page for more details on the changelog.\n\n https://github.com/k6io/k6/releases/tag/v!version!\"",
+    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Grafana Labs \n - Go to the release page for more details on the changelog.\n\n https://github.com/k6io/k6/releases/tag/v!version!\"",
```
Comment (olegbespalov) [inline packaging/rpm.json:5]:
```suggestion
    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Grafana Labs \n - Go to the release page for more details on the changelog.\n\n https://github.com/grafana/k6/releases/tag/v!version!\"",
```

File: packaging/deb.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "k6",
-    "maintainer": "Load Impact AB",
+    "maintainer": Raintank Inc. d.b.a. Grafana Labs,
     "description": "Load testing for the 21st century",
-    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Load Impact \n - Go to the release page for more details on the changelog.\n\n https://github.com/k6io/k6/releases/tag/v!version!\"",
+    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Grafana Labs \n - Go to the release page for more details on the changelog.\n\n https://github.com/k6io/k6/releases/tag/v!version!\"",
```
Comment (olegbespalov) [inline packaging/deb.json:5]:
```suggestion
    "changelog-cmd": "echo \"* $(date \"+%a %b %d %Y\") Grafana Labs \n - Go to the release page for more details on the changelog.\n\n https://github.com/grafana/k6/releases/tag/v!version!\"",
```

File: packaging/k6.portable.nuspec [hunk 3]
```diff
@@ -18,4 +18,4 @@
     <licenseUrl>https://github.com/k6io/k6/blob/master/LICENSE.md</licenseUrl>
```
Comment (olegbespalov) [inline packaging/k6.portable.nuspec:18]:
```suggestion
    <licenseUrl>https://github.com/grafana/k6/blob/master/LICENSE.md</licenseUrl>
```

**Review by sniku — APPROVED** [review]

File: packaging/deb.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "copyrights": [
       {
         "files": "*",
-        "copyright": "2020 Load Impact AB",
+        "copyright": 2022 Raintank Inc. d.b.a. Grafana Labs,
```
Comment (sniku) [inline packaging/deb.json:18]: looks like you missed quotes here `"`

File: packaging/deb.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "k6",
-    "maintainer": "Load Impact AB",
+    "maintainer": Raintank Inc. d.b.a. Grafana Labs,
```
Comment (sniku) [inline packaging/deb.json:3]: looks like you missed quotes here `"`

File: packaging/deb.json (hunk 2 above)
Comment (olegbespalov) [reply packaging/deb.json:5]: @codebien was it reverted, because of the force push? :thinking:

**Review by olegbespalov — APPROVED** [review]

**Review by na-- — APPROVED** [review]

**Review by sniku — APPROVED** [review]

**Pull request merged**